app/src/components/Alerts.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from 'react';
+import '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Do we need to import  bootstrap.css to each component or it could be injected 1 time in the index.js file?